LanguageFeatures/Wildcards/binding_A02_t03.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A local declaration whose name is `_` does not bind that name to
+/// anything. This means you can have multiple local declarations named `_` in
+/// the same namespace without a collision error.
+///
+/// @description Checks that it is not an error to have a multiple local
+/// declarations named `_`. Test parameters of typedefs.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=wildcard-variables
+
+import '../../Utils/expect.dart';
+
+typedef int F1(_, int _, var _);
+typedef int F2(_, int _, var _, [_, int _ = 0, var _ = ""]);
```

Comment:
I think it's a compile-time error for any formal parameter in a type alias to have a default value.

Reply:
But `dynamic` is nullable, so it should be enough to just omit the default value.

Reply:
(I think the default values are still there, and also that the compile-time errors caused by those default values would be unwanted.)

Reply:
> (I think the default values are still there, and also that the compile-time errors caused by those default values would be unwanted.)

Probably you are looking at the outdated diff. I don't see any default values on line 18 and other lines except 28 and  31. But these default values should be Ok

Reply:
OK, thanks!